pkg/cmd/get/get_applications.go

```diff
@@ -131,12 +131,17 @@ func (o *GetApplicationsOptions) Run() error {
 func (o *GetApplicationsOptions) generateTable(kubeClient kubernetes.Interface, list applications.List) table.Table {
 	table := o.generateTableHeaders(list)
 
+OUTER:
 	for _, a := range list.Items {
 		row := []string{}
 		name := a.Name()
 
 		for _, k := range o.sortedKeys(list.Environments()) {
 
+			if len(a.Environments) == 0 {
+				continue OUTER
```

Comment:
maybe just use a `break` here to reduce complexity

Reply:
@macox if I break that only affects the inner loop not the outer one, or am I missing something?